app/src/main/java/org/jellyfin/mobile/bridge/ExternalPlayer.kt

```diff
@@ -205,7 +167,8 @@ class ExternalPlayer(
     }
 
     private fun notifyEvent(event: String, parameters: String = "") {
-        if (event in arrayOf(Constants.EVENT_CANCELED, Constants.EVENT_ENDED, Constants.EVENT_TIME_UPDATE) && parameters == parameters.filter { it.isDigit() }) {
+        val allowedEvents = arrayOf(Constants.EVENT_CANCELED, Constants.EVENT_ENDED, Constants.EVENT_TIME_UPDATE)
```

Comment:
Should this be a companion object constant?

Reply:
I suppose it would be better, yes.